mod_reforged/config/math/math.nut

```diff
@@ -31,13 +31,81 @@
 		return (newDistance < oldDistance) ? _newValue : _oldValue;
 	}
 
-	function seededRand( _seed, _min, _max )
+	function seededRand( _min, _max, ... )
```

Comment:
An alternative name for this function could be `RandWithFixedSeed`
As of right now it sounds very similar to seedRandom